adminer/include/design.inc.php

```diff
@@ -22,6 +22,9 @@ function page_header($title, $error = "", $breadcrumb = array(), $title2 = "") {
 <meta http-equiv="Content-Script-Type" content="text/javascript">
 <meta name="robots" content="noindex">
 <meta name="referrer" content="origin-when-crossorigin">
+<?php if (!$error && isset($_GET["processlist"], $_GET["autorefresh"])) { ?>
```

Comment:
I'm not sure what you mean by this sorry. Are you saying that I should replace the `meta http-equiv` tag with a `Location` header?